index.css

```diff
@@ -0,0 +1,80 @@
+body {
+  font-family: Arial, sans-serif;
+  margin: 0;
+  padding: 0;
+  background-color: #f0f0f0;
+}
+
+.container {
+  max-width: 1000px;
```

Comment:
the unit size is px by default. you shouldn't write it

Reply:
it dose not work when i remove the px